topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
+> (Note: deleting a dataset in the history does not free quota until you purge it.)
+> 
+> ![Galaxy quota info](../../images/bgc_calib/galaxy_storage_handling.png)
+> 
+> - **Time limit for interactive tools**: 24h. After that, ODV closes automatically.  
+> - **Avoid weekends**: upgrades are performed and can stop tools/workflows.  
+> - **Max interactive tools simultaneously**: 2.
+{: .details}
+
+## Nitrate calibration
+
+> <comment-title>Oxygen prerequisite</comment-title>
+> Because oxygen concentration is implied in the estimation of the *nitrate adjustment from neural network method*,  
+> platforms to be calibrated must have **good-quality oxygen** (adjusted in real time or delayed mode for Argo floats or gliders).
+{: .comment}
+
+> <details-title>World Ocean Atlas reference</details-title>
+> Nitrate calibration needs a potential reference dataset such as the **WOA annual climatology**.  
+> You can [download it here](https://www.ncei.noaa.gov/thredds-ocean/catalog/woa23/DATA/nitrate/netcdf/all/1.00/catalog.html?dataset=woa23/DATA/nitrate/netcdf/all/1.00/woa23_all_n00_01.nc) and upload it into Galaxy.
 {: .details}
 
-# Manage your data 
+# Manage your data
+
+## Step 1: Get your data 
+
+> <hands-on-title>Get your data from the S3 server</hands-on-title>
+> - Click on **Upload** in the vertical panel on the left
+> - In the pop-up window, click on **Choose from repository**
+> - In the top search bar, search for `argo`
+> - In the result list, select **Argo marine floats data and metadata from Global Data Assembly Centre (Argo GDAC)**
+> - Navigate to `pub/dac/coriolis/4903881`
+> - Tick the following:
+>   - `4903881_meta.nc`
+>   - All files under `profiles/`
+> - Click on **Select** (bottom-right)
+> - Remove the synthetic files `SR4903881_*.nc` (by clicking the trash icon)
+> - Click on **Start**
+> - Wait until all files are stored in your history on the right (green color)
+>
+> ![S3 upload interface](../../images/bgc_calib/s3_upload.png)
+{: .hands_on}
 
-> <hands-on-title>Get data</hands-on-title>
-> Upload the argo data files (meta, core and BGC) of the float 4903881 from the S3 service. 
-> - 2a. Click on “Upload” on the vertical panel on the left. A pop-up is launched
-> - 2b. Click on “Choose from repository” at the bottom of the popup
+> <hands-on-title>Get your data from your computer</hands-on-title>
+> - Click on **Upload** in the vertical panel on the left
+> - In the pop-up, click on **Choose local file**
+> - Select the files on your system (Argo float or WOA)
+> - Click on **Open** then **Start**
+> - Wait until all files are stored in your history (green color)
 >
-> ![Image of how to uploade remote data files](../../images/bgc_calib/bgc_2a.png)
+> ![Local upload interface](../../images/bgc_calib/local_upload.png)
+{: .hands_on}
+
+## Step 2: Organize your data
+
+> <hands-on-title>Create a dataset collection (from empty history)</hands-on-title>

```

Comment:
this says from an empty history in the title, but the instructions assume the files are already in the history?